lib/Duo/API.pm

```diff
@@ -2,7 +2,7 @@ package Duo::API;
 use strict;
 use warnings;
 
-our $VERSION = '1.2.1';
+our $VERSION = '1.3.0';
```

Comment:
Do we want to combine adding the feature + the version bump in one PR?

Reply:
Because `duo_api_perl` doesn't have a "release process", I put them together so that master's behavior change and version change are "released" simultaneously. I was planning to "rebase and merge" to keep them as separate commits.

I can move the version bump into its own PR if that's preferred though.

Reply:
that's fine.  future us can create an actual release process